suite/tests/api/ir_aarch64.c

```diff
@@ -3960,6 +3960,71 @@ test_asimddiff(void *dc)
     test_instr_encoding(dc, OP_umull2, instr);
 }
 
+static void
+test_sys(void *dc)
+{
+    byte *pc;
+    instr_t *instr;
+
+    /* SYS #<op1>, <Cn>, <Cm>, #<op2>{, <Xt>}
+     *
+     * Data cache operations are aliases of SYS:
+     * DC <dc_op>, <Xt>
+     * is equivalent to
+     * SYS #<op1>, C7, <Cm>, #<op2>, <Xt>
+     */
+
+    /* DC ZVA, Xt => SYS #3, C7, C4, #1, Xt */
```

Comment:
It would be more convenient for the user, for (relatively) frequently constructed instructions like this, to have dedicated constructors (`INSTR_CREATE_dc_zva`?) or named constants for the immediates.